mobile/library/swift/Stream.swift

```diff
@@ -37,7 +37,7 @@ public class Stream: NSObject {
   ///                        callback.
   ///
   /// - returns: This stream, for chaining syntax.
-  public func readData(_ byteCount: size_t) -> Stream {
```

Comment:
This gets inferred as being `std.size_t` when building with C++ interop enabled, which is not what we want.